Source/Core/InputCommon/DynamicInputTextureConfiguration.h

```diff
@@ -0,0 +1,53 @@
+// Copyright 2019 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <map>
+#include <string>
+#include <vector>
+
+#include "Common/CommonTypes.h"
+#include "Common/IniFile.h"
+
+namespace InputCommon
+{
+class DynamicInputTextureConfiguration
+{
+public:
+  explicit DynamicInputTextureConfiguration(const std::string& json_file);
+  ~DynamicInputTextureConfiguration();
+  void GenerateTextures(const IniFile::Section* sec, const std::string& controller_name) const;
+
+private:
+  struct DynamicInputTextureData
+  {
+    struct ColorKey
+    {
+      u8 r;
+      u8 g;
+      u8 b;
+    };
+
+    std::string m_colorKeyImageName;
```

Comment:
```suggestion
    std::string m_color_key_image_name;
```

ditto for the next two member names